impl/doc/debug.md

```diff
@@ -25,8 +45,8 @@ are bound by respective formatting trait.
 
 E.g., for a structure `Foo` defined like this:
 ```rust
-use derive_more::Debug;
-
+# use derive_more::Debug;
+#
```

Comment:
I think I prefer these `use` statements to be shown, because this is the only derive where we shadow an already existing derive. Making that clear to users is important imho.

Reply:
Ohh... `rustdoc` somehow doesn't allow to expand such examples to see the hidden code, only mdBook does. Okay, I'll rollback these.

Reply:
Done in https://github.com/JelteF/derive_more/pull/381/commits/3b67c97295bae2018700da6dd3d763c4b24a5644.